File: w03/d03/james_liu/fibonacci/lib/fibonacci.rb
def sum_of_even_fibonaccis(limit)
    holder = [1,2]
    #sets array with the first 2 terms
    

    while (holder[-1] + holder[-2]) < limit
    #sets limit    
      holder << holder[-1] + holder[-2]
    #shoves sum of last 2 values into the holder array 
    end
    
    return holder.select { |num| num.even? }.reduce(:+)
    #selects only the even numbers and adds them together
end

File: w06/d05/alexander_levine/morning_ex_fibonacci/fibonacci.js
var fibonacciArray = [];

fibonacciArray.push(1);
fibonacciArray.push(1);
for (var i = 0; i < 8; i++) {
  fibonacciArray.push((fibonacciArray[fibonacciArray.length-1]) + (fibonacciArray[fibonacciArray.length-2]));
}
console.log(fibonacciArray);

var sum = 0

for (var j = 0; j < fibonacciArray.length; j++) {
  sum += fibonacciArray[j];
}

console.log(sum);
